### PR TITLE
fix(DB/Creature): Lich King Audio and Text during Chapter 3 of DK Questline

### PR DESCRIPTION
**Pull request by Voxstrasza**
## Changes Proposed:
-  Applies Path of Frost aura to Lich King during Phase 3 of the Ebon Hold questlines.  Spell ID is already applied to the phase 1 counterpart.
-  Add audio, create text group and apply SAI to Lich King during Phase 3 of Ebon Hold.  Will cycle through 17 voice lines over the course of 17 minutes (60 sec interval between voices).  The voicefiles are called A_EH_LichKing_Chapter3Shout01-17 to confirm that these are in the right place.

## SOURCE:
https://youtu.be/b863yUtcCzM?t=4920
https://youtu.be/kqqxXo9FEbI?t=3628

https://www.wowhead.com/npc=29110/the-lich-king quotes also list Battle of Light's Hope for some reason

## Tests Performed:
I applied the SQL and sat and wait while the npc went through the voice files.  There are 17 in total.

[With Aura](https://i.imgur.com/X3fl0Hj.jpg)

[Voice Files](https://i.imgur.com/e4IJgYM.jpg)

## How to Test the Changes:
.go c 130896
make sure aura is under his feet
make sure he cycles through text and voice lines

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1638655418843943400.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES (29110, 250, 14, 'The sacking of the Scarlet Enclave will be a lesson for history!', 14, 0, 100, 0, 0, 14790, 29504, 0, 'Lich King - Ebon Hold - Phase 3 Voice Lines');
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES (29110, 250, 15, 'There is nowhere to run...', 14, 0, 100, 0, 0, 14791, 29505, 0, 'Lich King - Ebon Hold - Phase 3 Voice Lines');
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES (29110, 250, 16, 'Let the destruction of this place serve as a lesson to all those who would dare oppose the Scourge!', 14, 0, 100, 0, 0, 14792, 29506, 0, 'Lich King - Ebon Hold - Phase 3 Voice Lines');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1638655418843943400.sql:26]: Please fix the sql style

Comment (Voxstrasza) [reply data/sql/updates/pending_db_world/rev_1638655418843943400.sql]: > 

fixed the formatting

File: data/sql/updates/pending_db_world/rev_1638655418843943400.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+/* LK Path of Frost Effect
+*/
+
+UPDATE `creature_addon` SET `auras`= 53274 WHERE `guid` = 130896;
+
+/* Lich King text + voice during phase 3 of DK questline
+*/
+
+DELETE FROM `creature_text` WHERE (`CreatureID` = 29110) AND (`GroupID` = 250);
```
Comment (claudiodfc) [inline data/sql/updates/pending_db_world/rev_1638655418843943400.sql:11]: Change delete query to this:

``` sql
DELETE FROM `creature_text` WHERE (`CreatureID` = 29110) AND (`GroupID` IN (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16));
```

**Review by claudiodfc — CHANGES_REQUESTED** [review]

**Comment by claudiodfc** [comment]
### Test Case
- Turn GM mode
- Executing command `.go c 130896`
- The Lich King does use his creature_text

**Comment by claudiodfc** [comment]
@azerothcore/developers can someone give a review?

**Comment by acidmanifesto** [comment]
> @azerothcore/developers can someone give a review?

Is his sql good or no? I am really not a sql guys really.

**Comment by claudiodfc** [comment]
Other than the change I pointed out, it looks good. Otherwise, you can't re-run the query.

**Comment by Voxstrasza** [comment]
> Other than the change I pointed out, it looks good. Otherwise, you can't re-run the query.

Should be G2G now.  I've been away.  Apologies.

**Review by acidmanifesto — APPROVED** [review]

**Pull request merged**